testsuite/features/support/twopence_init.rb

```diff
@@ -223,6 +220,9 @@ def file_inject(node, local_file, remote_file)
 # Other global variables
 $product = product
 $pxeboot_mac = ENV['PXEBOOT_MAC']
+$sle11sp3_terminal = ENV['SLE11SP3_TERMINAL']
```

Comment:
So you are making the env variables common between CI and BV, but not adding them in the nodes? Is this deliberate or are the nodes missing in lines 34, 104 etc?

Reply:
Does this fall in the node_by_host in 235 maybe so it's not needed?

Reply:
They are not twopence objects ("nodes"), they are simple strings. You can't reach them via SSH.